app.py

```diff
@@ -34,8 +34,62 @@ def get_temp_zip_dir():
 
 setup_log()
 
-st.sidebar.header("v3-beta")
-
+st.sidebar.header("v3-beta") 
+
+
+button_styles_css = """
+<style>
```

Comment:
Move it to index.css and access it over __init__.py of public with the variable index_css

`css.markdown(index_css, allow_unsafe_html=True)`

Reply:
Done